lib/eclipse/Parser/Parser.cpp

```diff
@@ -337,12 +340,20 @@ void ParserState::handleRandomText(const string_view& keywordString ) const {
     std::stringstream msg;
     std::string trimmedCopy = keywordString.string();
 
+
     if (trimmedCopy == "/") {
         errorKey = ParseContext::PARSE_RANDOM_SLASH;
         msg << "Extra '/' detected at: "
             << this->current_path()
             << ":" << this->line();
-    } else {
+    }
+    else if (lastSizeType == OTHER_KEYWORD_IN_DECK) {
+      errorKey = ParseContext::PARSE_EXTRA_RECORDS;
+      msg << "Too many records in keyword: " 
+          << lastKeyWord
```

Comment:
OK - hopefully last comment; please attach the file and line information to the error message in the same as has been done for the two error blocks.